var/spack/repos/builtin/packages/nest/package.py

```diff
@@ -14,9 +14,11 @@ class Nest(CMakePackage):
 
     homepage = "http://www.nest-simulator.org"
     url      = "https://github.com/nest/nest-simulator/releases/download/v2.12.0/nest-2.12.0.tar.gz"
+    url      = "https://github.com/nest/nest-simulator/archive/v3.0.tar.gz"
```

Comment:
This is redefining the attribute at line 16, which means it will invalidate the shas for other versions.

Reply:
The new `url` is the correct version, but do you happen to know if there's an automatic way of re-calculating shas?